docs/src/components/pricing/PricingTable.tsx

```diff
@@ -18,18 +18,17 @@ const planInfo = {
   community: {
     color: 'green',
     title: 'Community',
-    description:
-      'Get started with the industry-standard UI library for building React user interfaces (open-source, MIT licensed).',
+    description: 'Get started with the industry-standard UI library for building React user interfaces (open-source, MIT licensed).',
   },
   pro: {
     color: 'blue',
     title: 'Pro',
-    description: 'Best for professional developers building enterprise or data-rich applications.',
+    description: 'Best for developers building data-rich applications with essential features for handling a high volume of data.',
   },
   premium: {
     color: 'gold',
     title: 'Premium',
-    description: 'Unlock all the most advanced features and professional support.',
+    description: 'Best for developers building data-rich applications with the most advanced UX use cases.',
```

Comment:
Should we highlight here the Support upgrade you get with bumping the plan? I'm assuming that going to Premium gets me more than the components' features.

Reply:
Here's what I would suggest, building off of my suggestion for the Pro description.

```suggestion
    description: 'Upgrade to MUI X Premium for the most advanced features necessary for data-rich applications, as well as the highest priority for support.',
```